portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -74,52 +85,53 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String nickname = (String) entity.getProperty("nickname");
       String email = (String) userService.getCurrentUser().getEmail();
       ArrayList<String> messageReplies = (ArrayList) entity.getProperty("replies");
-      BlogMessage message = new BlogMessage(messageId, tag, comment, nickname, email, messageReplies, timestamp);
+      BlogMessage message = new BlogMessage(
+            messageId, tag, comment, nickname, email, messageReplies, timestamp);
       messages.add(message);
```

Comment:
Again does this compile at all? I don't know where we define "messages", do you mean "BlogMessages"?

Reply:
If this doesn't compile, please always ensure the code compiles before a commit. The code compiles (and actually runs) is one key assumption in the code base and also for reviewers.